core/src/main/scala/org/apache/spark/MapOutputTracker.scala

```diff
@@ -706,7 +706,7 @@ private[spark] class MapOutputTrackerWorker(conf: SparkConf) extends MapOutputTr
     val statuses = mapStatuses.get(shuffleId).orNull
     if (statuses == null) {
       logInfo("Don't have map outputs for shuffle " + shuffleId + ", fetching them")
-      val startTime = System.currentTimeMillis
```

Comment:
@MaxGekk, are there all instances of using `currentTimeMillis`? If so, let's also add a style rule in a separate PR to prevent `currentTimeMillis`. I know this is in the guide and think it's reasonable to add the rule if possible.